static/js/graphs_page.js

```diff
@@ -870,7 +870,7 @@ var graphPage = {
     },
     onTapGraphElement: function (evt) {
         // get target
-        var target = evt.cyTarget;
+        var target = evt.Target;
```

Comment:
@JingVT @adbharadwaj - `evt.Target` is incorrect. It should be `evt.target`
Refer - https://github.com/cytoscape/cytoscape.js/issues/1537